src/sentry/backup/comparators.py

```diff
@@ -666,9 +666,7 @@ def get_default_comparators():
     default_comparators: ComparatorMap = defaultdict(
         list,
         {
-            "sentry.apitoken": [
-                HashObfuscatingComparator("refresh_token", "token", "token_last_characters")
-            ],
+            "sentry.apitoken": [HashObfuscatingComparator("refresh_token", "token")],
```

Comment:
```suggestion
            "sentry.apitoken": [
                HashObfuscatingComparator("refresh_token", "token"),
                # The `ApiToken` model's `.save()` method ensures that this field matches the token.
                # Because this is field is behind an option, we want to avoid failing on JSON where it
                # is not set, so we just ignore the comparison, and defer to the token comparison
                # instead.
                IgnoredComparator("token_last_characters"),
            ],
```